packages/mroonga-package-task.rb

```diff
@@ -96,7 +96,7 @@ def detect_ubuntu_package_version(code_name, package_name)
       URI.open(all_packages_url) do |all_packages|
         all_packages.each_line do |line|
           case line
-          when /\A#{Regexp.escape(package_name)} \((.+?)[\s)]/o
+          when /\A#{Regexp.escape(package_name.sub(/-/, "-server-"))} \((.+?)[\s)]/o
```

Comment:
This will be difficult to understand in the future.

How about naming this value?

```ruby
ubuntu_package_name = package_name.sub(/-/, "-server-")
source_names = [...]
source_names.each do |...|
  ...
    ...
      when /\A#{Regexp.escape(ubuntu_package_name)} .../o
    ...
  ...
end
```

Reply:
Ah...
I see. I've fixed it at https://github.com/mroonga/mroonga/pull/417/commits/eda00db7225e03a0d3000fb7d56dab0a4717d933.